src/ufoLib2/objects/contour.py

```diff
@@ -147,7 +147,16 @@ def getControlBounds(
         """
         return getControlBounds(self, layer)
 
-    # XXX: Add property controlPointBounds (defcon compat API)?
+    @property
+    def controlPointBounds(self) -> Optional[BoundingBox]:
+        """Returns the (xMin, yMin, xMax, yMax) bounding box of the glyph,
+        taking only the control points into account.
+
+        Gives inaccurate results with extruding curvatures.
```

Comment:
What do you mean? what's extruding curvatures?

Reply:
Ah maybe you mean curves that don't have control points at their extremes. But I wouldn't characterize the result from the ControlBoundsPen as inaccurate. They are what they are, and curve is always contained within the control points' bounds (by the convex hull property), so they are correct. If one wants "tighter" bounds one can use the other `bounds` property.

Reply:
No idea 😄 I copied the doc string of the corresponding Contour functions.

Reply:
ah :) so the original text was incorrect /cc @madig

Reply:
i think i copied it from defcon, or maybe i didn't

Reply:
ok let's just remove the sentence

Reply:
Done